demos/python_demos/llm_text_generation/servable_unary/model.py

```diff
@@ -184,5 +187,5 @@ def generate():
             else:
                 completions = [a + b for a, b in zip(completions, partial_result)]
         print('end', flush=True)
-
-        return serialize_completions(batch_size, completions)
+        t1.join()
+        return serialize_completions(batch_size, completions, token_count[0])
```

Comment:
I'm not sure why are working on a numpy array, just to return a number in line 191. Then, in serialize_completions you create numpy array anyway `Tensor("token_count", np.array([token_count], dtype=np.int32))` @bstrzele